actionview/lib/action_view/helpers/tags/select.rb

```diff
@@ -15,7 +15,7 @@ def initialize(object_name, method_name, template_object, choices, options, html
 
         def render
           option_tags_options = {
-            selected: @options.fetch(:selected) { value.to_s },
+            selected: @options.fetch(:selected) { value || value.to_s },
```

Comment:
Let's avoid the usage of `||` here since it doesn't communicate the intention.

```suggestion
            selected: @options.fetch(:selected) { value.nil? ?  "" : value },
```